lib/ioki/apis/platform_api.rb

```diff
@@ -70,6 +70,17 @@ class PlatformApi
         except:      [:create, :update, :delete],
         model_class: Ioki::Model::Platform::Pause
       ),
+      Endpoints.crud_endpoints(
+        :line,
+        base_path:   [API_BASE_PATH, 'products', :id],
+        model_class: Ioki::Model::Platform::Line
+      ),
+      Endpoints.crud_endpoints(
+        :line_stop,
+        base_path:   [API_BASE_PATH, 'lines', :id],
```

Comment:
I guess `line_stops` are also part of the product resource. In that case you need to adjust the `base_path` to:
```ruby
base_path: [API_BASE_PATH, 'products', :id, 'lines', :id]
```

Reply:
Right, i had to push nested line_stops before, but changed it in the api after and did not apply here. Thanks.